src/app/containers/PageHandlers/withOptimizelyProvider/index.jsx

```diff
@@ -19,7 +40,7 @@ const withOptimizelyProvider = (Component, noUserId = false) => {
         isServerSide
         timeout={500}
         user={{
-          id: noUserId ? null : getAtUserId(),
+          id: hasUserId && getCknsMvt(),
```

Comment:
so if there is no userId then false is ok as a value here?

Reply:
Yep, if the userId is false the if statement in the ReactSDK for the provider in our case will evaluate correctly as it can handle any falsey value:

https://github.com/optimizely/react-sdk/blob/2731ec8cdfed33fae71584324babb5b371033eb4/src/Provider.tsx#L48

Reply:
We've changed the implementation now, so it will in fact return null as we have removed the boolean parameter from you're suggestion